features/step_definitions/files_steps.rb

```diff
@@ -26,7 +41,6 @@
 end
 
 
-Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |file_name, content|
-  expect(uncommitted_files).to eql [file_name]
-  expect(IO.read file_name).to eql content
+Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |name, content|
```

Comment:
Thanks for sharing, I wanted to ask you for your solution, but then got stuck in an emergency situation at work for the last 2 days. Having such a global command-line search and replace solution sounds very useful to have! The boilerplate for this has always made me not use such a solution regularly. But it should be super easy to write an easier to use bash wrapper, like `gr foo bar`, which globally replaces all occurrences of "foo" with "bar" in the current directory or its children. I'll add that to my tool belt!